### ISSUE TITLE
llvm: 'atomic' file not found

### ISSUE THREAD

**Issue by dornED** [Linuxbrew/homebrew-core#1190]
I can't build llvm on CentOS 6.8. According to the logs

https://gist.github.com/c0e0d0fcd9c2427fc4493beef8f36310

the error is: <code>fatal error: 'atomic' file not found</code>

**Comment by sjackman** [comment]
```
make[5]: Entering directory `/scratch/dornbier/brew/.tmp/llvm-20161111-5449-n9pxmd/tools/clang/runtime/compiler-rt-bins'
[ 94%] Building CXX object lib/scudo/CMakeFiles/clang_rt.scudo-x86_64.dir/scudo_allocator.cpp.o
/scratch/dornbier/brew/.tmp/llvm-20161111-5449-18zpsqm/llvm-3.9.0.src/projects/compiler-rt/lib/scudo/scudo_allocator.cpp:27:10: fatal error: 'atomic' file not found
#include <atomic>
         ^
1 error generated.
make[5]: *** [lib/scudo/CMakeFiles/clang_rt.scudo-x86_64.dir/scudo_allocator.cpp.o] Error 1
make[5]: Leaving directory `/scratch/dornbier/brew/.tmp/llvm-20161111-5449-n9pxmd/tools/clang/runtime/compiler-rt-bins'
```
https://gist.githubusercontent.com/anonymous/c0e0d0fcd9c2427fc4493beef8f36310/raw/9c64746238e8696f0c98a589016986e53acd3276/02.make

**Comment by sjackman** [comment]
Thanks for the bug report, @dornED. I'm afraid that I don't have a fix for you right now. I'll leave this issue open, and hopefully someone else will be able to provide more information.

**Comment by maxim-belkin** [comment]
Finally I've figured out the fix for Linuxbrew (#1202)

**Comment by dornED** [comment]
Awesome! I just tried your fix, and it works fine for me.

Many thanks!

**Comment by maxim-belkin** [comment]
Closing this as this issue has been addressed in #1202
